<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovingWidgetClass</class>
 <widget class="QWidget" name="MovingWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MovingWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PTZ马达移动测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up_pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="left_pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>左</string>
   </property>
  </widget>
  <widget class="QPushButton" name="right_pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>右</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down_pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>成功</string>
   </property>
  </widget>
  <widget class="QPushButton" name="false_pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>失败</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
